pythontabcmd2/tabcmd2_controller.py

```diff
@@ -0,0 +1,21 @@
+import argparse
+import sys
+from .parsers.parent_parser import ParentParser
+from .logger_config import get_logger
+
+
+logger = get_logger('pythontabcmd2.tabcmd2_controller', 'info')
+
+
+class Tabcmd2Controller:
```

Comment:
is there a specific reason this is in a separate class to main()? Seems like they could be combined for a tiny bit less complexity in reading the code.